install/openshift-web-console-operator/install.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- name: IMAGE
+  value: openshift/origin:latest
+- name: NAMESPACE
+  # This namespace must not be changed.
+  value: openshift-core-operators
+- name: LOGLEVEL
+  value: "0"
+- name: COMPONENT_LOGLEVEL
+  value: "0"
+- name: COMPONENT_IMAGE
+  value: openshift/origin-web-console:latest
+- name: NODE_SELECTOR
+  value: "{}"
+objects:
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: openshiftwebconsoleconfigs.webconsole.operator.openshift.io
+  spec:
+    scope: Cluster
+    group: webconsole.operator.openshift.io
+    version: v1alpha1
+    names:
+      kind: OpenShiftWebConsoleConfig
+      plural: openshiftwebconsoleconfigs
+      singular: openshiftwebconsoleconfig
+
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    namespace: ${NAMESPACE}
+    name: openshift-web-console-operator
+    labels:
+      app: openshift-web-console-operator
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: openshift-web-console-operator
+    template:
+      metadata:
+        name: openshift-web-console-operator
+        labels:
+          app: openshift-web-console-operator
+      spec:
+        serviceAccountName: openshift-web-console-operator
+        containers:
+        - name: operator
+          image: ${IMAGE}
+          imagePullPolicy: IfNotPresent
+          command: ["hypershift", "experimental", "openshift-webconsole-operator"]
+          args:
+          - "-v=${LOGLEVEL}"
+        nodeSelector: "${{NODE_SELECTOR}}"
```

Comment:
Double braces?

Reply:
> Double braces?

that makes the template processor substitute as an integer (well bare, but integer in this case)